frontend/src/components/ProfileDropdown/ProfileDropdown.tsx

```diff
@@ -0,0 +1,27 @@
+import classes from './ProfileDropdown.module.scss'
+import { Award, Handshake, LogOut, Map } from 'lucide-react'
+import ProfileDropDownItems from '../ProfileDropdownItems/ProfileDropdownItems'
+
+
+function ProfileDropdown (props: {username: string}) {
+  return (
+    <div className={classes.dropDownMenu}>
+      <h3 className={classes.username}>{props.username}</h3>
+      
+      <ProfileDropDownItems href='/' text='Levels'>
```

Comment:
nitpick but would make sense to call this 'ProfileDropDownItem' because the component only represents one item